source/B2B.Transactions.CimMessageAdapter/MarketActivity/MarketActivityRecordForwarder.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Text;
+using System.Threading.Tasks;
+using System.Transactions;
+using Azure.Messaging.ServiceBus;
+using Energinet.DataHub.MarketRoles.Infrastructure.Serialization;
+
+namespace B2B.CimMessageAdapter.MarketActivity
+{
+    public class MarketActivityRecordForwarder : IMarketActivityRecordForwarder, IDisposable, IAsyncDisposable
+    {
+        private readonly IJsonSerializer _jsonSerializer;
+        private readonly string _connectionString = "<Connection String>";
+        private readonly string _queueName = "<Queue name";
+        private bool _disposed;
+        private TransactionScope? _transactionScope;
+        private ServiceBusClient? _client;
+        private ServiceBusSender? _serviceBusSender;
+
+        public MarketActivityRecordForwarder(IJsonSerializer jsonSerializer)
+        {
+            _transactionScope = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled);
+            _client = new ServiceBusClient(_connectionString);
+            _serviceBusSender = _client.CreateSender(_queueName);
+            _jsonSerializer = jsonSerializer;
+        }
```

Comment:
```suggestion
        private ServiceBusSender _serviceBusSender;

        public MarketActivityRecordForwarder(IJsonSerializer jsonSerializer, ServiceBusSender sender)
        {
            _transactionScope = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled);
            _serviceBusSender = sender
            _jsonSerializer = jsonSerializer;
        }
```

Could consider prepping this for DI? (something like above?)